SQL/database_changelog.txt

```diff
@@ -1,3 +1,18 @@
+
+20th July 2017, by Shadowlight213
+Added role_time table to track time spent playing departments.
+Also, added flags column to the player table.
+
+CREATE TABLE `role_time` ( `ckey` VARCHAR(32) NOT NULL , `job` VARCHAR(128) NOT NULL , `minutes` INT UNSIGNED NOT NULL, PRIMARY KEY (`ckey`, `job`) ) ENGINE = InnoDB;
+
+ALTER TABLE `player` ADD `flags` INT NOT NULL AFTER `accountjoindate`;
+
+UPDATE `schema_revision` (`major`, `minor`) VALUES (3, 1);
```

Comment:
This is incorrect syntax, you want:
``UPDATE `schema_revision\` SET minor = 1``

I'm not sure if it'll complain about not using safe updates, i'll check later.